docs/deployment.md

```diff
@@ -0,0 +1,248 @@
+  Members of these roles will be able to visit the dashboard only if they are TA for particular course (see `ta_roles`). 
+
+If both `access_dashboard_for_all_orgs_groups` and `access_dashboard_role_groups` are empty or missing, admin dashboard 
+is effectively disabled.
+
+Example configuration:
+
+    "XBLOCK_SETTINGS": {
+      "group_project_v2": {
+        "dashboard_details_url": "/admin/workgroup/dashboard/programs/{program_id}/courses/{course_id}/projects/{project_id}/details?activate_block_id={activity_id}",
+        "ta_review_url": "/courses/{course_id}/group_work/{group_id}?activate_block_id={activate_block_id}",
+        "access_dashboard_for_all_orgs_groups": ["mcka_role_mcka_admin"],
+        "access_dashboard_groups": ["mcka_role_client_admin", "mcka_role_internal_admin"],
+        "access_dashboard_ta_groups": ["mcka_role_mcka_ta"],
+        "ta_roles": ["assistant"]
+      }
+    }
+    
+    
+## Enabling Group Project XBlock v2 in a course
+
+To enable the use of Group Project XBlock v2 in the course:
+
+1. Open the course in Studio.
+2. In the top menu choose Settings -> Advances Settings
+3. Add `gp-v2-project` to "Advanced Module List"
+4. Optionally, add `ooyala-player` to "Advanced Module List" - this player is used for video resources
+
+As a result, `Advanced Module List` option should look like that (provided there are no other advanced modules are 
+enabled for course): 
+![Advanced Module List Image](/docs/images/advanced_module_list.png)
+
+
+## (Optional) Notifications integration
+
+Group Project XBlock v2 sends a number of notifications via edx-notifications app. In order to make those notifications
+appear correctly, make sure `NOTIFICATION_CLICK_LINK_URL_MAPS` Django setting contains the following record:
+
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}'
+    
+If you're using edx-solutions/edx-platform fork, appropriate value is already set. Otherwise, note that 
+`NOTIFICATION_CLICK_LINK_URL_MAPS` are read from `lms.env.json`, so they should be modified there.    
+
+Example:
+
+```
+NOTIFICATION_CLICK_LINK_URL_MAPS = {
+    'open-edx.studio.announcements.*': '/courses/{course_id}/announcements',
+    'open-edx.lms.leaderboard.*': '/courses/{course_id}/cohort',
+    'open-edx.lms.discussions.*': '/courses/{course_id}/discussion/{commentable_id}/threads/{thread_id}',
+    'open-edx.xblock.group-project.*': '/courses/{course_id}/group_work?seqid={activity_location}',
+    'open-edx.xblock.group-project-v2.*': '/courses/{course_id}/group_work?activate_block_id={location}',
+}
+```  
+
+# External/API dependencies of GWv2 
+
+Below is a list of a non-standard dependencies of student-facing functionalities of GWv2. 
+
+## XBlock services
+
+* `settings` GWv2 extensively uses settings defined in `XBLOCK_SETTINGS` in edx lms configuration. It uses
```

Comment:
I think that this fails into "External Dependency" of GWv2, not strictly non-standard one. Should we leave it on that basis.